src/java.base/share/conf/security/java.security

```diff
@@ -583,6 +583,9 @@ sun.security.krb5.maxReferrals=5
 #   UsageConstraint:
 #       usage [TLSServer] [TLSClient] [SignedJAR]
 #
+#   TLSCipherConstraint:
```

Comment:
These comments may need to be in the tls property, adding that these are specific constraints for TLS operations.   I realize `UsageConstraints` say TLS in them, but they are used for multiple properties.   Maybe the tls property should follow the layout of the jar property, where it lists the options and refers to the meanings in certpath.  Except for the new property.

Reply:
Yeah, it can be somewhat confusing. The thing is `jdk.tls.disabledAlgorithms` section has the following instruction:
`# See the specification of "jdk.certpath.disabledAlgorithms" for the
# syntax of the disabled algorithm string.
`
So it looks like the `jdk.certpath.disabledAlgorithms` section is used as the central location for constraint description. Thus, I've decided to put the description for this new constraint there. I'll modify the description to follow the layout of the jar property as suggested.